transformations/negate_strengthen/transformation.py

```diff
@@ -0,0 +1,456 @@
+import random
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import spacy
+from spacy.tokenizer import Tokenizer
+import pandas as pd
+import nltk
```

Comment:
It seems that nltk is not in the main requirements.txt file, so you might need to put nltk in this transformation's requirements.txt file.

Reply:
Maybe pandas as well

Reply:
Ok, done!

Reply:
Actually I think nltk and pandas were somehow included due to other packages' dependencies, but anyway, I included them explicitly now! Thanks!